### PR TITLE
Ports long-range analyzer from bee 2: electric boogaloo

### PR DESCRIPTION
**Pull request by Tulenich**
## About The Pull Request

Adds a long-range analyzer with a _**brand new awful sprite!**_

## Why It's Good For The Game

It's a useful tool or something idk kuberoot was asking for it

## Changelog
:cl:
add: A new long-range analyzer with an awful sprite
tweak: A long-range analyzer now spawns in the CE's toolbelt instead of a regular one
tweak: A long-range analyzer is now researchable and craftable in engi- and scilathe
/:cl:

### CODE REVIEW

**Review by MaltVinegar — CHANGES_REQUESTED** [review]
I am able to scan things that are out of sight

**Comment by Tulenich** [comment]
Fine, I'll try to fix it

File: singulostation/code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+			to_chat(user, "<span class='info'>Plasma: [round(plasma_concentration*100, 0.01)] % ([round(environment.get_moles(/datum/gas/plasma), 0.01)] mol)</span>")
+
+		for(var/id in environment.get_gases())
+			if(id in GLOB.hardcoded_gases)
+				continue
+			var/gas_concentration = environment.get_moles(id)/total_moles
+			to_chat(user, "<span class='alert'>[GLOB.meta_gas_info[id][META_GAS_NAME]]: [round(gas_concentration*100, 0.01)] % ([round(environment.get_moles(id), 0.01)] mol)</span>")
+		to_chat(user, "<span class='info'>Temperature: [round(environment.return_temperature()-T0C, 0.01)] &deg;C ([round(environment.return_temperature(), 0.01)] K)</span>")
+
+/proc/los_check(mob/user, atom/target)
```
Comment (MaltVinegar) [inline singulostation/code/game/objects/items/devices/scanners.dm:58]: A. This doesn't even work and B. why not use an already existing LoS check?

**Review by MaltVinegar — CHANGES_REQUESTED** [review]

**Comment by MaltVinegar** [comment]
OP doesn't want to do it anymore

**Pull request closed without merging**